index.bs

```diff
@@ -5413,7 +5413,8 @@ partial interface MLGraphBuilder {
         - <dfn>input</dfn>: an {{MLOperand}}. The input 4-D tensor.
         - <dfn>options</dfn>: an optional {{MLResample2dOptions}}. The optional parameters of the operation.
 
-    **Returns:** an {{MLOperand}}. The output 4-D tensor.
+    **Returns:**
+        - an {{MLOperand}}. The output 4-D tensor.
```

Comment:
(nit) Same line or next line?

```suggestion
    **Returns:** an {{MLOperand}}. The output 4-D tensor.
```
Otherwise it's inconsistent with pad above:

    **Returns:** an {{MLOperand}}. The padded output tensor. Each dimension of the output tensor can be calculated as follows:

Reply:
I commented on Slack that about half the spec keeps "Returns: ..." on one line, the other half splits the "Returns" across two lines. If you have a preference for one line I'm happy to put up a PR to do that everywhere.

Reply:
🤔 Hmm, it's reasonable to put them on the same line currently because all current operators either return a single operand anyway, or they (e.g. split and gru) return multiple operands are homogeneous (basically an array of tensors). If we return multiple heterogenous outputs in the future though, it would make sense to list them as bullet point items on the next line, consistently with the "Arguments" and pending "Data types" sections. Well, I don't have a strong preference currently - I just noticed it because those differing lines were near each other in the CR.